plugin/src/main/java/com/denizenscript/denizen/events/entity/ExpBottleBreaksScriptEvent.java

```diff
@@ -0,0 +1,81 @@
+package com.denizenscript.denizen.events.entity;
+
+import com.denizenscript.denizen.events.BukkitScriptEvent;
+import com.denizenscript.denizen.objects.EntityTag;
+import com.denizenscript.denizencore.objects.ObjectTag;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.containers.ScriptContainer;
+import com.denizenscript.denizencore.utilities.CoreUtilities;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.entity.ExpBottleEvent;
+
+public class ExpBottleBreaksScriptEvent extends BukkitScriptEvent implements Listener {
+
+    // <--[event]
+    // @Events
+    // exp bottle breaks

```

Comment:
Please name the event (and class and etc) `experience bottle` in line with
- Mojang's naming conventions
- Denizen's no-random-shorthands convention

Spigot's janky names shouldn't be followed.